docs/src/reference/temporian/operators/window/moving_quantile.md

```diff
@@ -0,0 +1 @@
+::: temporian.EventSet.moving_quantile
```

Comment:
Awesome.
Can you also edit: docs/src/reference/index.md

Reply:
* added to the list of windows operations.